packages/wix-ui-tpa/src/components/Dialog/Dialog.tsx

```diff
@@ -19,7 +19,9 @@ export interface DialogProps extends TPAComponentProps {
   /** Children to render inside dialog */
   children?: React.ReactNode;
   /** A class name that will be added to the content wrapper */
-  contentClassName?: string;
+  contentWrapperClassName?: string;
```

Comment:
Oh, sorry for the confusion, I didn't mean changing the original class name, cause this is a breaking change.

```suggestion
  contentClassName?: string;
```